internal/render/container.go

```diff
@@ -236,13 +236,23 @@ func probe(p *v1.Probe) string {
 	return on
 }
 
+func MakeContainerRes(po *v1.Pod, isInit bool, index int) ContainerRes {
+	return ContainerRes{
+		pod:    po,
+		IsInit: isInit,
+		Index:  index,
+		Age:    po.GetCreationTimestamp(),
+	}
+}
+
 // ContainerRes represents a container and its metrics.
 type ContainerRes struct {
```

Comment:
All that should be needed here is to change IsInit->idx and pass this in from the call site.